apps/charterafrica/src/components/Datasets/useDatasets.js

```diff
@@ -4,11 +4,14 @@ import queryString from "@/charterafrica/utils/datasets/queryString";
 
 const fetcher = (url) => fetch(url).then((res) => res.json());
 
-function useDatasets(query, pathname) {
+function useDatasets(query, pathname, locale) {
```

Comment:
`locale` should be part of `query` & should be handled by `queryString`. See articles' `queryString` implementation: https://github.com/CodeForAfrica/ui/blob/045bd61ba810c1d34ee3b7e11fc689da8635c00c/apps/charterafrica/src/utils/articles/queryString.js#L12